proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1124,20 +1107,12 @@ private SymbolScopeSuper classStateSuper(JPNode classNode, String className) {
       // vendor libraries, etc.
       return null;
     }
-    PUB pub = new PUB(refSession, FileStuff.fullpath(file));
-    boolean pubIsCurrent = pub.loadTo(PUB.HEADER);
     ParseUnit pu = new ParseUnit(file, refSession);
-    pu.setPUB(pub);
     JPNode superClassTree = (JPNode) classNode.getLink(IConstants.SUPER_CLASS_TREE);
     try {
       if (superClassTree != null) {
         pu.setTopNode(superClassTree);
       } else {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00108)